src/pages/benefits/viewAllBenefit/ApplicationDetails.tsx

```
@@ -66,6 +69,9 @@ const ApplicationDetails: React.FC = () => {
     "approved" | "rejected"
   >();
 
+  const [amountDetail, setAmountDetail] = useState<Record<string, any> | null>(
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)